### PR TITLE
More realistic birds

### PR DESCRIPTION
**Pull request by 8street**
#### Summary

SUMMARY: Balance "More realistic birds"

#### Purpose of change

I noticed, when I poking JSON file, that not all flags, morale and aggression correspond to the real behavior of the birds. Also, some birds do not have the correct weight and volume.

#### Describe the solution

I added the missing flags and changed birds weight and volume. 
For example, geese and turkeys are a little more aggressive than they were.
All birds now have the flag FEATHER.  
Also some birds, like geese, crows, turkeys, chickens are now not so much afraid of humans. They can now start running away from the player within a few tiles from him. Before my changes, they ran away as soon as they saw the player, about 20-30 tiles.

#### Testing

Spawning crows, ducks, geese, turkeys, chickens, grouses, pheasants and see their behavior.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
The feather flag is deprecated and doesn't do anything.

**Comment by 8street** [comment]
Thank you. I'd deleted FEATHER flag. And deleted from doc too.

**Comment by anothersimulacrum** [comment]
If you're willing to make some minor C++ changes, it should be deleted from mtype.h and monstergenerator.cpp too.

**Comment by 8street** [comment]
This is outside the scope of this PR. I'll create a new PR soon.

File: data/json/monsters/bird.json [hunk 1]
```diff
@@ -111,7 +114,11 @@
     "volume": "5 L",
     "weight": "3750 g",
     "hp": 15,
+    "aggression": 1,
```
Comment (actual-nh) [inline data/json/monsters/bird.json:117]: Perhaps add PLAYER_CLOSE as an anger trigger, but HURT as a fear trigger?

Comment (8street) [reply data/json/monsters/bird.json:117]: Sorry, I didn't have time to answer you, before merge. PLAYER_NEAR_BABY and perhaps HURT flags are good here. You can create new PR, if you want.

Comment (actual-nh) [reply data/json/monsters/bird.json:117]: OK, will take a look at that (need to grade some lab reports before I do anything involved... sigh).

**Pull request merged**